src/gpu/kmeans/kmeans_labels.h

```diff
@@ -72,23 +62,6 @@ static const char *cudaGetErrorEnum(cublasStatus_t error)
 }
 #endif
 
-inline cublasStatus_t throw_on_cublas_error(cublasStatus_t code, const char *file,
-                                       int line) {
-
-
-  if (code != CUBLAS_STATUS_SUCCESS) {
-    fprintf(stderr,"cublas error: %s %s %d\n", cudaGetErrorEnum(code), file, line);
-    std::stringstream ss;
-    ss << file << "(" << line << ")";
-    std::string file_and_line;
-    ss >> file_and_line;
-    thrust::system_error(code, thrust::cuda_category(), file_and_line);
-  }
-
-  return code;
-}
-
-
 extern cudaStream_t cuda_stream[MAX_NGPUS];
```

Comment:
I don't like this global variable. Consider making this a member of the GPUInfo singleton or removing if it's no longer used.

Reply:
That's not part of kmeans||. If I change that, it might be in another PR that deals with kmeans in general. Is it acceptable for you?